host/src/pack/dpkg/etc/rc.mwc

```diff
@@ -12,6 +12,23 @@ do
     chmod 666 /dev/shm/looking-glass$i
 done
 
+echo "Set sleep-inhibitor if not already done..."
+if [ ! -d /usr/local/share/sleep-inhibitor/plugins/is-wakelock-active ]
+then
+    cp /opt/cfc/mwc/bin/is-wakelock-active /usr/local/share/sleep-inhibitor/plugins/
+fi
+if [ ! -d /etc/sleep-inhibitor.conf ]
+then
+   cp /opt/cfc/mwc/bin/sleep-inhibitor.conf /etc/
```

Comment:
maybe we need 4 space?

Reply:
This should be fine. It is same as the above one